text/3392-leadership-council.md

```diff
@@ -72,13 +72,13 @@ To reduce the size of this RFC, non-binding reference materials appear in separa
 
 # 動機
 
-The Rust project consists of hundreds of globally distributed people, organized into teams with various purviews. However, a great deal of work falls outside the purview of any established team, and still needs to get done.
+Rust 專案由分布在全球的數百人組成，組織成擁有不同職責範圍的團隊。然而，許多工作超出了任何既有團隊的範圍，且仍然需要完成。
```

Comment:
- Based on [definition from `hundreds of`](https://tw.dictionary.search.yahoo.com/search?p=hundreds%20of), I wonder it doesn't mean 數百人 here; instead, it means `無數` or `許多` 🤔?
- we should still translate "職責範圍" for purview in the second sentence instead of just "範圍"?
- "needs to get done" should include "被" in translation?

how about this 👇 ?

```suggestion
Rust 專案是由無數遍佈在全球各地的人所組成，並組織成擁有不同職責範圍的團隊。然而，還有許多工作超出了任何既有團隊的職責範圍，且仍然需要被完成。
```